File: C++/1460. Make Two Arrays Equal by Reversing Subarrays.cpp
class Solution {
public:
    bool canBeEqual(vector<int>& target, vector<int>& arr) {
        short target_size = target.size(), arr_size = arr.size();
        if (target_size != arr_size)
            return false;
        short c[1001] = { 0 };
        for (short i = 0; i < target_size; ++i)
            ++c[target[i]];
        for (short i = 0; i < target_size; ++i)
            --c[arr[i]];
        for (short i = 1; i < 1001; ++i)
            if (c[i] != 0)
                return false;
        return true;
    }
};

File: JAVA/1460. Make Two Arrays Equal by Reversing Subarrays.java
class Solution {
    public boolean canBeEqual(int[] target, int[] arr) {
       Arrays.sort(target);
       Arrays.sort(arr);
       int j=0;
       for(int i=0;i<arr.length;i++)
       {
          if(target[i]==arr[j])
          {
              j++;
              continue;
          }
          else
          {
              return false;
          }
       } 
       return true;
    }
}
